deepmd/pd/train/training.py

```
@@ -99,6 +103,11 @@ def __init__(
         Args:
         - config: The Dict-like configuration with training options.
         """
+        from paddle.distributed import fleet
```

Ruff (0.12.2)

Redefinition of unused `fleet` from line 30

Remove definition: `fleet`

(F811)